screeps-game-api/src/objects/macros.rs

```diff
@@ -86,3 +86,33 @@ macro_rules! impl_room_object_properties {
         )*
     );
 }
+
+
+// Macro for mass implementing `StructureProperties`, `PartialEq` and `Eq` for a type. 
+// 
+// This macro accepts a comma-separated list of types on which to implement the unsafe `StructureProperties` trait on
+// a screeps object. 
+// From that implementation, the type gets the `id` method which is used to implement `PartialEq` and `Eq`.
+// 
+// # Example
+// ```
+// macro_rules!(OwnedStructure, Structure, StructureContainer)
```

Comment:
Unless I'm missing something, I think this line was meant to be something else?

Reply:
The copy/paste monster is real!